sdk/sui-owned-object-pools/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "commonjs",
+    "lib": ["dom", "esnext"],
+    "moduleResolution": "node",
+    "declaration": true,
+    "rootDir": "src",
+    "outDir": "dist",
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "composite": false,
+    // output .js.map sourcemap files for consumers
+    "sourceMap": true
+  },
+  "include": ["src/**/*"]
```

Comment:
I think you'll probably want to set up references to sui.js. Our build scripts also require a second tsconfig for the esm build (I would copy the two tsconfig files from one of the other SDKs)